web/components/EventBlueprint.js

```diff
@@ -1,74 +1,79 @@
 /**
- * Copyright (c) 2018, salesforce.com, inc.
+ * Copyright (c) 2020, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or
  * https://opensource.org/licenses/BSD-3-Clause
  */
 
+import React from 'react';
 import PropTypes from 'prop-types';
+import moment from 'moment';
 import './chat.css';
 
-const moment = require('moment');
-const React = require('react');
-
-class EventBlueprint extends React.Component {
-  render() {
-    let name;
-    const { event } = this.props;
+/**
```

Comment:
i would say its fine to just have these in prop types and a note to see there rather than keep putting them all in notes for any place props are passed in